src/app/component/MissionComponent/AcceptMissionButton.jsx

```diff
@@ -4,11 +4,11 @@ import CheckIcon from "@material-ui/icons/Check";
 import Mission from "../../model/Mission";
 
 /**
- * @param {userUID} the user UID that accept the mission
+ * @param {handleUpdatedMissions} function that updates missions tied to a user.
```

Comment:
we dont use this function anymore ? Maybe we should do as Paul comment and remove this now that we have everything refractor to the model system

Reply:
Correct -- I removed it because I thought you said you wanted to have reusable components and so we moved the volunteer/mission-state tracking to the firebase redux store.